app/lib/features/news/widgets/news_post_editor/selected_action_button.dart

```diff
@@ -123,4 +142,29 @@ class SelectedActionButton extends ConsumerWidget {
           },
         );
   }
+
+  Widget linkActionButton(BuildContext context, RefDetails refDetail) {
+    final uri = refDetail.uri();
+    if (uri == null) return SizedBox.shrink();
+    return SizedBox(
+      width: 300,
+      child: Card(
+        child: ListTile(
+          leading: const Icon(Atlas.link),
+          onTap: () => openLink(uri, context),
+          title: Text(
+            refDetail.title() ?? L10n.of(context).unknown,
+            maxLines: 2,
+            overflow: TextOverflow.ellipsis,
+            style: Theme.of(context).textTheme.labelMedium,
+          ),
+          subtitle: Text(
+            uri,
+            maxLines: 2,
+            overflow: TextOverflow.ellipsis,
+          ),
```

Comment:
May I suggest we hightlight the title rather than the url?
```suggestion
          title: Text(
            refDetail.title() ?? L10n.of(context).unknown,
            maxLines: 2,
            overflow: TextOverflow.ellipsis,
          ),
          subtitle: Text(
            uri,
            maxLines: 1,
            overflow: TextOverflow.ellipsis,
            style: Theme.of(context).textTheme.labelMedium,
          ),
```